sections/03_categorize.md

```diff
@@ -617,10 +647,14 @@ Despite the difficulties and uncertainties, machine learning practitioners (and
 particularly those who use deep neural networks, which are challenging to
 interpret) must remain cognizant of these dangers and make every effort to
 prevent harm from discriminatory predictions. To reach their potential in this
-domain, deep learning methods will need to be interpretable. Researchers need to
+domain, deep learning methods will need to be interpretable (more in the discussion section). Researchers need to
 consider the extent to which biases may be learned by the model and whether or
 not a model is sufficiently interpretable to identify bias. We discuss the
 challenge of model interpretability more thoroughly in Discussion.
+<<<<<<< HEAD
+
+=======
+>>>>>>> greenelab/master
```

Comment:
Looks like you committed some lines from a partially resolved conflict. Delete these lines?

Reply:
Yes. Should I do it?

Reply:
@qiyanjun yes, please remove these extra lines.  GitHub is now showing new conflicts with several files so we'll have to resolve those as well before we can merge this.